docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ New ENV vars:
 - `ADVISORY_LOCK_CHECK_INTERVAL` (default: 1s) - when advisory locking mode is enabled, this controls how often Chainlink checks to make sure it still holds the advisory lock. It is recommended to leave this at the default.
 - `ADVISORY_LOCK_ID` (default: 1027321974924625846) - when advisory locking mode is enabled, the application advisory lock ID can be changed using this env var. All instances of Chainlink that might run on a particular database must share the same advisory lock ID. It is recommended to leave this at the default.
 - `LOG_FILE_DIR` (default: chainlink root directory) - if `LOG_TO_DISK` is enabled, this env var allows you to override the output directory for logging.
+- `SHUTDOWN_GRACE_PERIOD` (default: 15s) - when node is shutting down gracefully and exceeded this grace period, it terminates immediately (trying to close DB connection) to avoid being SIGKILLed. 
```

Comment:
Where does `15s` come from? Should we derive it from the docker default of `10s`? https://docs.docker.com/compose/reference/stop/
And we may need to aim for _less_ than that, to be sure we actually complete in time (accounting for execution time, and possibly unbounded scheduling delay). Even significantly less, like 5s, since this is something that is meant to work in worst cases as well. (we could hide this nuance from the users by waiting for some % of what is set, like we do in cltest [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/internal/cltest/cltest.go#L928-L929))

Reply:
Initially I had 10s, but then CI integration tests started to fail. I increased to 15s and now they pass. I do understand that ideal value should be around 5s. If need to check if I can set this value to 15s just for integration tests (is this Okay for integration tests?), and then we can keep 5s as production default value.

Reply:
:thinking:

Reply:
I think it's ok to set it differently for tests if we need to, but we should try to understand why.